client/test/e2e/inventory/inventory_list.spec.js

```diff
@@ -0,0 +1,98 @@
+
+  let currentDate = new Date();
+  let uniqueIdentifier = currentDate.getTime().toString();
+
+  // inventory list items
+  let metadata = {
+    code  : uniqueIdentifier,
+    text  : '[E2E] Inventory Article',
+    price : 7.57,
+    group : 'Test inventory group',
+    type  : 'Article',
+    unit  : 'Act',
+    unit_weight : 1,
+    unit_volume : 1
+  };
+
+  let metadataUpdate = {
+    code : uniqueIdentifier.concat('_updated'),
+    text : '[E2E] Inventory Article updated',
+    price : 7.77,
+    group : 'Inventory Group',
+    type  : 'Service',
+    unit  : 'Pill',
+    unit_weight : 7,
+    unit_volume : 7
+  };
+
+  it('Successfully creates a new inventory item (metadata)', () => {
+    FU.buttons.create();
+    FU.input('$ctrl.session.label', metadata.text);
+    FU.input('$ctrl.session.code', metadata.code);
+    element(by.model('$ctrl.session.consumable')).click();
+    FU.input('$ctrl.session.price', metadata.price);
+    FU.select('$ctrl.session.group', metadata.group);
+    FU.select('$ctrl.session.type', metadata.type);
+    FU.select('$ctrl.session.unit', metadata.unit);
+    FU.input('$ctrl.session.unit_weight', metadata.unit_weight);
+    FU.input('$ctrl.session.unit_volume', metadata.unit_volume);
+    FU.buttons.submit();
+    components.notification.hasSuccess();
+  });
+
+  it('Successfully updates an existing inventory item (metadata)', () => {
+    element(by.css('[data-edit-metadata="' + metadata.code + '"]')).click();
+    FU.input('$ctrl.session.label', metadataUpdate.text);
+    FU.input('$ctrl.session.code', metadataUpdate.code);
+    element(by.model('$ctrl.session.consumable')).click();
+    FU.input('$ctrl.session.price', metadataUpdate.price);
+    FU.select('$ctrl.session.group', metadataUpdate.group);
+    FU.select('$ctrl.session.type', metadataUpdate.type);
+    FU.select('$ctrl.session.unit', metadataUpdate.unit);
+    FU.input('$ctrl.session.unit_weight', metadataUpdate.unit_weight);
+    FU.input('$ctrl.session.unit_volume', metadataUpdate.unit_volume);
+    FU.buttons.submit();
+    components.notification.hasSuccess();
+  });
+
+  it('Dont creates a new inventory item (metadata) for invalid data', () => {
+    FU.buttons.create();
+    FU.input('$ctrl.session.label', metadata.text);
+    FU.input('$ctrl.session.unit_weight', metadata.unit_weight);
+    FU.input('$ctrl.session.unit_volume', metadata.unit_volume);
+    FU.buttons.submit();
+
+    // check validations
+    FU.validation.ok('$ctrl.session.label');
+    FU.validation.ok('$ctrl.session.unit_weight');
+    FU.validation.ok('$ctrl.session.unit_volume');
+    FU.validation.error('$ctrl.session.code');
+    FU.validation.error('$ctrl.session.price');
+    FU.validation.error('$ctrl.session.group');
+    FU.validation.error('$ctrl.session.type');
+    FU.validation.error('$ctrl.session.unit');
+
+    // be sure not success
+    expect(element(by.css('[data-notification-type="notification-success"]')).isPresent())
```

Comment:
The problem with Travis is that you never close the modal before starting the next test in the series!  This modal is present on the page when it tries to use the Posting Journal, which breaks...

You need to add an action that closes the modal.